llpc/util/llpcUtil.h

```diff
@@ -114,4 +115,26 @@ inline unsigned log2(T u) {
   return logValue;
 }
 
+// Returns true if shaderInfo has the information required to compile an unlinked shader of the given type.
+bool hasDataForUnlinkedShaderType(Vkgc::UnlinkedShaderStage type,
+                                  llvm::ArrayRef<const Vkgc::PipelineShaderInfo *> shaderInfo);
+
+// Returns the shader stage mask for all shader stages that can be part of the given unlinked shader type.
+unsigned int getShaderStageMaskForType(Vkgc::UnlinkedShaderStage type);
+
+// Returns the name of the given unlinked shader type.
+const char *getTypeName(Vkgc::UnlinkedShaderStage type);
```

Comment:
Maybe `getStageTypeName`? `TypeName` alone sounds ambiguous to me, I would not associate it with shader stages as my first thought